src/app/services/export-excel/excel-export-sample-1/excel-export-sample-1.component.ts

```diff
@@ -27,6 +30,12 @@ export class ExcelExportSample1Component implements OnInit {
 
   public ngOnInit(): void {
     this.data = INVOICE_DATA;
+    this.expr = [
+        { dir: SortingDirection.Asc, fieldName: "ShipCountry", ignoreCase: false,
+          strategy: DefaultSortingStrategy.instance() },
+        { dir: SortingDirection.Asc, fieldName: "ShipCity", ignoreCase: false,
+          strategy: DefaultSortingStrategy.instance() }
```

Comment:
`ignoreCase` and `strategy` are not required properties.
```suggestion
        { dir: SortingDirection.Asc, fieldName: "ShipCountry" },
        { dir: SortingDirection.Asc, fieldName: "ShipCity" }
```